src/main/java/core/basesyntax/model/figure/Figure.java

```diff
@@ -1,13 +1,36 @@
 package core.basesyntax.model.figure;
 
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+import javax.persistence.Table;
+
+@MappedSuperclass
+@Table(name = "figure")
 public class Figure {
+    @Id
+    @GeneratedValue
```

Comment:
let's choose the strategy here

Reply:
added strategy = GenerationType.IDENTITY